railties/test/application/rake/multi_dbs_test.rb

```diff
@@ -215,11 +215,11 @@ def db_test_prepare_name(name, schema_format)
           animals_tables = lambda { rails("runner",  "-e", "test", "p AnimalsBase.lease_connection.tables").strip }
 
           if name == "primary"
-            assert_equal ["schema_migrations", "ar_internal_metadata", "books"].sort, JSON.parse(ar_tables[]).sort
+            assert_equal ["ar_internal_metadata", "books", "schema_migrations"].sort, JSON.parse(ar_tables[]).sort
             assert_equal "[]", animals_tables[]
           else
             assert_equal "[]", ar_tables[]
-            assert_equal ["schema_migrations", "ar_internal_metadata", "dogs"].sort, JSON.parse(animals_tables[]).sort
+            assert_equal ["ar_internal_metadata", "dogs", "schema_migrations"].sort, JSON.parse(animals_tables[]).sort
```

Comment:
Can you make these consistent with the other tests being changed? If the order in the array is correct, the `sort` on the expected value is not needed.

Reply:
Oops, good catch!